app/Global.java

```diff
@@ -0,0 +1,38 @@
+import java.net.URL;
+
+import play.*;
```

Comment:
This entire file might not even be necessary. This was part of the original solution to attach CORS headers to all responses, but I ended up implementing it differently later. We need to test if the app still works without Global.java, PreFlight.java, and with the preflight route removed from conf/routes